platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/wiring/PlatformWiringTests.java

```diff
@@ -87,7 +88,9 @@ void testBindings() {
                 .withEventCreationManager(mock(EventCreationManager.class))
                 .withInOrderLinker(mock(InOrderLinker.class))
                 .withConsensusEngine(mock(ConsensusEngine.class))
-                .withConsensusEventStream(mock(ConsensusEventStream.class));
+                .withConsensusEventStream(mock(ConsensusEventStream.class))
+                .withConsensusEngine(mock(ConsensusEngine.class))
```

Comment:
another incorrectly resolved merge conflict here

```
.withConsensusEngine(mock(ConsensusEngine.class))
```

is duplicated

Reply:
How this one got through is leaving me head scratching.  This wouldn't have compiled. There was a commit sitting on my local branch that hadn't been pushed.  I wonder if I pushed the merge without pushing the fixes afterward.

Reply:
verified by running locally and hitting break point on method triggered by heartbeat

Reply:
`.withConsensusEventStream(mock(ConsensusEventStream.class))` and `.withConsensusEngine(mock(ConsensusEngine.class))`

are still both duplicated

Reply:
deleting duplicates.